CONTRIBUTING.md

```diff
@@ -37,381 +48,295 @@ to sign the CLA. Therefore,
-contributing to the Knative code base, we've defined the roles of documentation
-contributors separately from the roles of code contributors.
-
-If you're looking for code contributor roles, see
-[ROLES](https://github.com/knative/community/blob/master/ROLES.md).
-
-### Member
-
-Established contributor to the Knative docs.
+The
+[Knative Documentation Working Group](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation)
+meets biweekly on Tuesdays at 9:30am PT.
+[Click here](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com)
+to see the exact dates on the Knative working group calendar.
 
-Members are continuously active contributors in the community. They can have
-issues and PRs assigned to them, might participate in working group meetings,
-and pre-submit tests are automatically run for their PRs. Members are expected
-to remain active contributors to the Knative documentation.
+The Working Group meetings are used to discuss documentation strategy and
+policy; we expect individual PR review to happen mostly asynchronously via
+GitHub, and [Slack](https://slack.knative.dev).
 
-All members are encouraged to help with PR reviews, although each PR must be
-reviewed by an official [Approver](#approver). In their review, members should
-be looking for technical correctness of the documentation, adherence to the
-[style guide](https://developers.google.com/style/), good spelling and grammar
-(writing quality), intuitive organization, and strong documentation usability.
-Members should be proficient in at least one of these review areas.
+# Workflow
 
-### Requirements
+There are roughly two workflows: writing / contributing new content, and
+refining and reviewing existing content.
 
-- Has made multiple contributions to the project or community. Contributions
-  might include, but are not limited to:
+## New Content
 
-  - Authoring and reviewing PRs on GitHub in the Docs or Website repos.
+We expect most new content to be written by the subject matter expert (SME)
+which would be the contributor who actually worked on the feature or example.
 
-  - Filing and commenting on issues on GitHub.
+When writing new content, keep the following in mind:
 
-  - Contributing to working group or community discussions.
+- Write one [type of documentation](#type of documentation) per page.
 
-- Subscribed to
-  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev).
+- If possible, follow the style/template of other documents of the same type.
 
-- Actively contributing to 1 or more areas.
+- Focus mostly on technical correctness; structure and language should be
+  roughly correct, but don't need heavy review in this phase.
 
-- Sponsored by 1 approver.
+The goal of adding new content is to get technically correct documentation into
+the repo before it is lost. Tech Writers may provide some quick guidance on
```

Comment:
In line what my comment above, the current set of writers are not familiar with the product yet and so this stage is just a copy edit. And historically, merge content rarely gets revisited until some opens an issue or another PR.

That said. I think we should be clear about expectations here.

Reply:
See my previous comment -- I think our previous process where tech writers spend a lot of time trying to teach engineers to improve the content isn't working very well, and we'd do better to try to revisit the pattern and move to something where the interface between tech writing and engineering isn't a long review chain on an engineer's PR.

By having engineering SMEs from the appropriate WG review these docs, we're taking tech writers out of the loop on the initial "content visit", and allowing them to focus on the "revisit content" aspect.